src/main/java/org/ebyhr/trino/storage/operator/SsvPlugin.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ebyhr.trino.storage.operator;
+
+import com.google.common.base.Splitter;
+import org.ebyhr.trino.storage.StorageColumnHandle;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.UncheckedIOException;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+
+public class SsvPlugin
+        implements FilePlugin
+{
+    private static final String DELIMITER = ";";
```

Comment:
We could merge this with `CsvPlugin` and `TsvPlugin` if we made `DELIMITER` a constructor parameter instead. Happy to do that if you want.

Reply:
Yes please, it's getting noisy.

Reply:
Merged impls: fb5c26be34d359faf57217bb84cf2dff60181922

(Accidentally replied to the wrong thread here..)